packages/react-components/src/atoms/__tests__/Link.test.tsx

```diff
@@ -37,6 +37,17 @@ describe.each`
     expect(color).toBe(paper.rgb);
   });
 
+  it('respects the underline prop', () => {
+    const { getByRole } = render(
+      <Link underline={false} href={href}>
+        text
+      </Link>,
+      { wrapper },
+    );
+    const { textDecoration } = getComputedStyle(getByRole('link'));
+    expect(textDecoration).toBe('none');
```

Comment:
First render without the prop and `expect(textDecoration).toMatchInlineSnapshot()` or so, then rerender with the prop set and to this :)

Reply:
It looks like we can't have `toMatchInlineSnapshot` twice on the same test.

Reply:
Yes you can o.O any error or so?

Reply:
`Jest: Multiple inline snapshots for the same call are not supported.`